src/meadowflow/event_log.py

```diff
@@ -51,10 +57,7 @@ class EventLog:
     PersistedEventLog)
     """
 
-    def __init__(self, event_loop: asyncio.AbstractEventLoop) -> None:
```

Comment:
Can't comment on the line directly but the comment on line 47 can be deleted now!